regression/cbmc/field-sensitivity-trace-wrong-counterexample-1/test.desc

```diff
@@ -0,0 +1,13 @@
+CORE
+test.c
+--trace
+^EXIT=10$
+^SIGNAL=0$
+^VERIFICATION FAILED$
+--
+x=\{ \.x=0 \}
```

Comment:
I originally had this as a different check (checking that there _was_ a non-zero assignment to `x.x`) but the `cprover-smt2` run didn't produce a valid trace at all which made this test fail.